test/utils/IexecWrapper.ts

```diff
@@ -518,81 +513,43 @@ export class IexecWrapper {
  * Extract address of a newly created entry in a registry contract
  * from the tx receipt.
  * @param receipt contract receipt
- * @param registryInstanceAddress address of the registry contract
  * @returns address of the entry in checksum format.
  */
 async function extractRegistryEntryAddress(
     receipt: ContractTransactionReceipt | null,
-    registryInstanceAddress: string,
 ): Promise<string> {
     if (!receipt) {
         throw new Error('Undefined tx receipt');
     }
-    const eventName = 'Transfer';
-    const eventAbi = [
-        'event Transfer(address indexed from, address indexed to, uint256 indexed tokenId)',
-    ];
-    const events = extractEventsFromReceipt(receipt, registryInstanceAddress, eventName, eventAbi);
-    if (events.length === 0) {
+    const event = extractEventFromReceipt(
+        receipt,
+        WorkerpoolRegistry__factory.createInterface(),
+        'Transfer',
+    ) as any as TransferEvent.OutputObject;
+    if (!event) {
         throw new Error('No event extracted from registry tx');
     }
     // Get registry address from event.
-    const lowercaseAddress = ethers.zeroPadValue(
-        ethers.toBeHex(BigInt(events[0].args.tokenId)),
-        20,
-    );
+    const lowercaseAddress = ethers.zeroPadValue(ethers.toBeHex(BigInt(event.tokenId)), 20);
     // To checksum address.
     return ethers.getAddress(lowercaseAddress);
 }
 
 /**
  * Extract a specific event of a contract from tx receipt.
  * @param txReceipt
- * @param address
+ * @param contractInterface
  * @param eventName
- * @param eventAbi
- * @returns array of events or empty array.
+ * @returns event
  */
-function extractEventsFromReceipt(
+function extractEventFromReceipt(
     txReceipt: ContractTransactionReceipt,
-    address: string,
+    contractInterface: Interface,
```

Comment:
Just curious whether we can use just the ABI of the event (`Fragment`) instead of all the contract's interface.

Reply:
ABI of the event would we totally fine yes :+1: